smartsim/_core/utils/telemetry/collector.py

```diff
@@ -238,16 +121,15 @@ class DBCollector(Collector):
     """A base class for collectors that retrieve statistics from an orchestrator"""
 
     def __init__(self, entity: JobEntity, sink: Sink) -> None:
-        """Initialize the collector"""
         super().__init__(entity, sink)
         self._client: t.Optional[redisa.Redis[bytes]] = None
-        self._address = _Address(
+        self._address = _DBAddress(
             self._entity.config.get("host", ""),
             int(self._entity.config.get("port", 0)),
         )
 
     async def _configure_client(self) -> None:
```

Comment:
in `def _configure_client`, why is it okay to continue even though a client connection was not successful? I notice that there are checks like `if not self._client: log."unable to collect" but should it be stopped at the root?

Reply:
Oop just read your reply to another comment on Orch not setting up yet! ignore